package.json

```
@@ -53,7 +53,7 @@
     "lodash": "^4.17.13",
     "react": "^16.8.6",
     "react-dom": "^16.8.6",
-    "styled-components": "^4.4.1"
+    "styled-components": "^5.1.0"
```

```suggestion
    "styled-components": ">=4 <6"
```

이런 식으로 해주면 사용하는 쪽에서 styled 버전을 결정할 수 있게 해줘서 새 버전에 대한 소프트랜딩이 가능할듯
(단, class101-ui에서 4와 5버전 둘 다 사용 가능하게 구현해야함)

---

여러 semver 표현식은 아래 링크 참고...
https://www.npmjs.com/package/semver

---

(제대로 쓴 건지 몰겠네)